lib/Runtime/Language/JavascriptExceptionOperators.cpp

```diff
@@ -62,7 +67,11 @@ namespace Js
 
     bool JavascriptExceptionOperators::CrawlStackForWER(Js::ScriptContext& scriptContext)
     {
-        return Js::Configuration::Global.flags.WERExceptionSupport && !scriptContext.GetThreadContext()->HasCatchHandler();
+        // If we have exception handling, but no user exception handling, we should make
+        // the additional backtrace information available, as that increases consistency
+        // between native and nonative approaches (native code has some automatic error-
+        // handling structures that get added)
```

Comment:
I've added a test in the new commit. The reason that none of our baselines were affected is that they pretty much all catch their exceptions in user code; this bug relates to errors _not_ caught in user code.